task3.py

```diff
@@ -0,0 +1,22 @@
+# Задайте список из вещественных чисел. Напишите программу, которая найдёт разницу 
+# между максимальным и минимальным значением дробной части элементов.
+# Пример:  - [1.1, 1.2, 3.1, 5, 10.01] => 0.2
+import random
+random.random()
+
+number = int(input('Задайте длину списка '))
+lst = []
+for i in range(number):
+    lst.append(round(random.uniform(1, 10), 3))
+print(lst)
+min = round(lst[0] % 1, 3)
+max = round(lst[1] % 1, 3)
+for i in range(0, number):
```

Comment:
```suggestion
for i in range(number):
```
Если вы собираетесь начинать сначала и не собираетесь указывать шаг, то ноль указывать не обязательно.